README.md

```diff
@@ -3,118 +3,28 @@ WooYellowCube allow the synchronization between WooCommerce and YellowCube (Swis
 
 Swiss Post offers an all-in logistics solution for distance selling with YellowCube. The range of services covers goods receipt, storage, picking, packaging, fast shipping, and returns management.
 
-Requirements: WooCommerce 3.0+, PHP 7
+Requirements: WooCommerce 3.5+, PHP 7
```

Comment:
dbDelta() was added in 3.5. Source: https://codex.wordpress.org/Creating_Tables_with_Plugins#Creating_or_Updating_the_Table